adapter_pipelines/ss2_single_sample/adapter_example_static_demo.json

```diff
@@ -8,7 +8,8 @@
   "AdapterSmartSeq2SingleCell.reference_bundle": "bf51d668-3e14-4843-9bc7-5d676fdf0e01",
   "AdapterSmartSeq2SingleCell.format_map": "gs://hca-dcp-mint-test-data/adapters/file_format_map.json",
   "AdapterSmartSeq2SingleCell.method": "SmartSeq2SingleCell",
-  "AdapterSmartSeq2SingleCell.schema_version": "5.1.0",
+  "AdapterSmartSeq2SingleCell.schema_version": "7.0.0",
+  "AdapterSmartSeq2SingleCell.analysis_file_version": "5.2.1",
```

Comment:
This I think is the best solution for us at this time. 

However, this is still painful for us. Per offline discussion, in case that major metadata schema version updates break our prod pipelines, we pin to specific versions here, so we only have to go back and update the code here when we are notified to do it, this requires other boxes (e.g. Purple) have the same behavior as we do (Pin to a specific version instead of always use the `latest` metadata version) and only update to new major versions coordinately. 

Ideally, as a consumer of the metadata, we should be able to use an abstract interface (API/CLI) instead of having to hardcode the parsing on our side and update frequently, as others are already discussing in the channels.

Reply:
At the moment, ingest will accept submissions that follow older schema versions so it should accept this format even if they update this again. But they may change that behavior in the future and only accept and validate against the latest schema.